Gemfile

```diff
@@ -7,7 +7,9 @@ gem "rails", "~> 4.2.7.1"
 gem "pg", "~> 0.15"
 
 gem "sass-rails", "~> 5.0"
-gem "bootstrap-sass"
+# gem "bootstrap-sass"
+gem 'bootstrap', '~> 4.0.0.alpha4'
```

Comment:
This is a bit unfortunate, but I guess we have to live with it for now. 😀 

Please use double quotes to match the other gem declarations, and delete the commented out `bootstrap-sass` gem line. (There are very few situations where we'd want commented out code in our codebase.)